numpy/core/_add_newdocs.py

```diff
@@ -4415,7 +4415,7 @@
     Examples
     --------
     >>> np.geterrobj()  # first get the defaults
-    [10000, 0, None]
+    [8192, 521, None]
```

Comment:
This documentation was incorrect, and caused by the doctests running before circular imports resolved (!?)